### PR TITLE
fix: make useTodos simpler to use

### PR DESCRIPTION
**Pull request by KristianKjerstad**
## Why is this pull request needed?
Make useTodos simpler to use. Now, the user don't have to remember the order of things returned from useTodos

## What does this pull request change?
make useTodos simpler to use

## Issues related to this change:
none

### CODE REVIEW

File: web/src/hooks/useTodos.tsx [hunk 1]
```diff
@@ -83,7 +83,7 @@ const useTodos = (): [
       .finally(() => setLoading(false))
   }
 
-  return [todos, isLoading, addItem, removeItem, toggleItem, error]
+  return { todos, isLoading, addItem, removeItem, toggleItem, error }
```
Comment (KristianKjerstad) [inline web/src/hooks/useTodos.tsx:86]: by returning an object instead of a list, the user of the hook don't have to remember the order of things returned

**Review by eoaksnes — APPROVED** [review]
Sweet, that it nice

File: web/src/hooks/useTodos.tsx (hunk 1 above)
Comment (sebastianvitterso) [reply web/src/hooks/useTodos.tsx:86]: ... and they don't have to "get" the `todos`-property in order to get the `isLoading`-property, which is equally as important. They can get the properties they want (via `{ isLoading } = useTodos()`)

**Review by sebastianvitterso — APPROVED** [review]
Nice price

**Pull request merged**